contracts/serverless-v2/Executors.sol

```diff
@@ -89,15 +89,15 @@ contract Executors is
      * @param _admin The address of the admin.
      * @param _images Array of enclave images to initialize.
      */
-    function initialize(address _admin, EnclaveImage[] memory _images) public initializer {
+    function initialize(address _admin, EnclaveImage[] memory _images, uint8 _env) public initializer {
         if (_admin == address(0)) revert ExecutorsZeroAddressAdmin();
 
         __Context_init_unchained();
         __ERC165_init_unchained();
         __AccessControl_init_unchained();
         __UUPSUpgradeable_init_unchained();
         __AttestationAuther_init_unchained(_images);
-        __TreeUpgradeable_init_unchained();
+        __EnvTreeUpgradeable_init_unchained(_env);
```

Comment:
doesn't make sense, what is this one single _env? How is it going to work? Fix it.